examples/todolist/tests/bindings/test_todolist.kts

```diff
@@ -21,3 +21,9 @@ assert(todo.getLast() == "Test Ünicode hàndling without an entry can't believe
 val entry2 = TodoEntry("Test Ünicode hàndling in an entry can't believe I didn't test this at first 🤣")
 todo.addEntry(entry2)
 assert(todo.getLastEntry().text == "Test Ünicode hàndling in an entry can't believe I didn't test this at first 🤣")
+
+assert(todo.getEntries().size == 5)
+
+todo.addEntries(listOf(TodoEntry("foo"), TodoEntry("bar")))
+assert(todo.getEntries().size == 7)
```

Comment:
Can we have a test verifying the entries from `getEntries`? Like comparing the values of the entries returned themselves

Reply:
Also maybe a tests  of lists something that is not a Record, not a big deal thou since I think we'll have to audit our testing anyways eventually 🙃